net/ampr-ripd/patches/000-Makefile.patch

```diff
@@ -0,0 +1,17 @@
+--- ampr-ripd-2.4/Makefile	2017-05-29 01:50:48.000000000 -0400
```

Comment:
Why we should have this patch? Please check OpenWrt guidelines.  It is not possible to disable MAN pages while compiling package?

Reply:
This patch does exactly that, it alters the distributed Makefile to not include man pages.

Reply:
I will attempt to do so, but as this may take time to resolve, will this be a no-go item?